vm/src/exceptions.rs

```diff
@@ -71,7 +71,10 @@ fn exception_str(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     );
     let type_name = objtype::get_type_name(&exc.typ());
     let msg = if let Some(m) = exc.get_attr("msg") {
-        objstr::get_value(&m)
+        match vm.to_pystr(&m) {
+            Ok(msg) => msg,
+            _ => "".to_string(),
```

Comment:
In this case it looks like we want the string ```"<exception str() failed>"```:

```
Traceback (most recent call last):
  File "test.py", line 7, in <module>
    assert False, Foo()
AssertionError: <exception str() failed>
```

Reply:
I don't think I got it correctly but I am assuming instead of `""` we need 
`"<exception str() failed>"`

Let me know if that is the case

Reply:
Yes, exactly.

Reply:
Done